### DISCUSSION TITLE
Sentence. Monika's opinion on tulpa.

### DISCUSSION THREAD

**Discussion by kirillPro2020** [Monika-After-Story/MonikaModDev#7911] [issue discussions]

**Comment by LooseEthics** [comment]
Since the creation process consists of inducing persistent hallucinations in yourself via forced cognitive dissonance, I'd say she'd be apprehensive and worried at best.

**Comment by multimokia** [comment]
Agreed. It seems like something she'd advise against or at least be worried about you delving into purely because of what it has you bordering on in the process.

**Reply by Hakan-Ekinci** [reply]
#- [x] Telefon

**Comment by lunulae** [comment]
I'd definitely be interested in a topic like this, but I agree that she might be skeptical at best.

There seems to be a small but growing internet culture around the creation of tulpas that resemble favorite characters, which I think she might be very concerned about as well. 
Any skepticism aside, I think Monika'd be firm that she _really_ doesn't want us to try to make one that resembles her. Best case scenario, such a thing would be creating a totally separate being meant to pretend to be her. She might understand the temptation to do so, to have a version of her that is closer to our perceived reality, but she'd probably discourage that as a selfish impulse.

**Comment by Abc2557** [comment]
I’ll say, tulpas and the subject of them is very controversial, from what I’ve seen, and I feel that there would be no way to please everyone when it comes to writing a topic on it. Although I would love a topic on it, I feel that there would be no way to write a topic that would be fine for singlets strongly against tulpas, system members, and those indifferent on the matter alike.

**Comment by kirillPro2020** [comment]
You can only write a general concept of tulpa. This is essentially an imaginary friend.

**Comment by ashyashton** [comment]
As someone who has something similar (though "tulpa" isn't the word I use, I just don't really have a label for it) I would say this would be really difficult. Because I don't know if Monika _would_ react well, and that would suck to hear from her. Plus, if this were implemented, there would have to be two versions, because there has to be a way for a person to say they _already_ fit into this category. And, if we're going with the assumption that Monika _would_ react badly, what's the protocol for that? What do we do in the case of Monika being against it but the player already has it? Does she get upset? Does she change her opinion? Is she just generally uncomfortable with the whole thing but doesn't say it because she doesn't want to offend the player? If so on that last one, that could lead to some really unhealthy communication issues, which is something that, in my opinion, should be avoided.

Overall, as nice as it would be to be recognized, I don't think there would be a good way to do this without upsetting one person or another, and it's really unlikely to do this in a satisfying way.

**Comment by multimokia** [comment]
Basing off some experience in the past that I've seen. It's very *very* easy to become rather unstable as a result of having a Tulpa.

With this in mind, Monika would likely want the player to try to stay away from forming/maintaining one. If the player has one, she'd likely disapprove of you having one and try to encourage you stop/disassociate from one (probably by mentioning some of the issues which can be as a result of having one). Purely for the sake of your own health.

I don't think it's a topic you can get away with not-offending people with, but from the purely factual standpoint, it's pretty dangerous to try and go down that route and I think Monika would rather you err on the side of caution.

**Comment by SamanthaLind** [comment]
I think when it comes to chosing if player does it or not we could do similar dialogue too smoking. When player tells her about doing something she doesn't like she says she's concerned for player's health and advice to us to stop doing it. But she doesn't mock player or is mean because of that, she's just concerned and remind that she wants what's best for player. I feel like something similar would fit here, with similar time and message. 
And before option to choose she could just describe this topic and say something about her feelings about it. Just like in smoking topic since both are quite similar. About player doing something that's not really something Monika likes.

**Comment by kirillPro2020** [comment]
I think if the player is happy and it does not threaten him and others. Let this person deceive himself, but he is happy. (this is the choice of everyone)

**Comment by Abc2557** [comment]
Just thinking out loud here, not trying to argue, but imo, a system is kind of like a marriage, except for the tulpa didn’t exist prior to said “marriage.” Marriages have their downsides, and not all marriages work out well. That doesn’t mean people think all marriages are bad, and the downsides don’t keep all people from getting married. Tulpa Systems are pretty similar. Sure, there can be downsides to getting a tulpa and becoming part of a system, but most tulpas exist to benefit the host, especially if they were created by traumatic events. Sure, those tulpas created to deal with trauma might have more side effects than a tulpa created for other purposes, but their main goal is to keep the host from having to bear all that weight. 
Tulpas don’t exist to harm the host, unless the host completely forced the tulpa to harm them. Even so, the tulpa can resist if they know it’s not good for them. Some tulpas will only do what the host asks, others will do only what they feel or know is best for the host, and others are just a mix in between. 
I’m not trying to argue, but I’m just trying to say that, despite the side effects, most hosts find it worthwhile to have a tulpa or tulpas in their life. (This is from a combination of experiences/opinions of my own, experiences/opinions of other systems, and knowledge I’ve gained just from doing research throughout the tulpa community. What I’ve said here doesn’t all apply to the rest of the plurality community, but some parts may also apply to other types of plurality)

(Again, I was just thinking out loud, so if it’s a bit sloppy, I apologize)
